apps/api/src/providers/provider.regression.ts

```diff
@@ -141,14 +157,79 @@ describe('Regression test - Providers', () => {
               _organizationId: session.organization._id,
               providerId,
               channel,
-              credentials: { apiKey: secrets.apiKey, secretKey: secrets.secretKey },
+              credentials: {
+                ...secrets,
+                from: FROM_EMAIL,
+              },
               active: true,
               deleted: false,
             });
           });
 
           it('should create notification template properly', async () => {
-            expect(1).to.eql(0);
+            template = await createRegressionNotificationTemplate(session, providerId);
+            expect(template.name).to.eql('regression-template-email');
+            expect(template.active).to.eql(true);
+            expect(template.steps[0].template.type).to.eql(channel);
+          });
+        });
+
+        describe('Triggering event use cases', () => {
+          beforeEach(async () => {
+            const { accountId, inboxId } = getMailtrapSecrets();
+            const inbox = await mailtrapService.cleanInbox(accountId, inboxId);
+
+            expect(inbox).to.deep.include({
+              id: inboxId,
+              name: 'Regression',
+              status: MailtrapInboxStatusEnum.ACTIVE,
+              emails_count: 0,
+              emails_unread_count: 0,
+            });
+          });
+
+          it('should send the notification to the chosen subscriber without attachment', async () => {
+            const eventTime = Date.now();
+            const notification = await triggerEvent(session, subscriber.subscriberId);
+
+            expect(notification.acknowledged).to.eql(true);
+            expect(notification.status).to.eql('processed');
+            expect(notification.transactionId).to.be.a('string');
+
+            await session.awaitRunningJobs(template._id, false, 0);
+
+            const { accountId, inboxId } = getMailtrapSecrets();
+            const inboxMessages = await mailtrapService.pollInbox(accountId, inboxId);
+
+            expect(inboxMessages.length).to.eql(1);
+            const [message] = inboxMessages;
+
+            const messageSentAt: Date = parseISO(message.sent_at);
+
+            expect(differenceInSeconds(eventTime, messageSentAt)).to.be.lessThan(MINUTE_IN_SECONDS / 2);
```

Comment:
This check is just in case some weird things happen and some old message is returned as we just call the inbox and get returned the existing emails in it.